### PR TITLE
Adding more unsupported features to readme

### PR DESCRIPTION
**Pull request by GuyMcComb**

### CODE REVIEW

**Review by swapnilraj — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -135,6 +135,13 @@ Please see the list below:
 |                  ternary operator                  |    :question:     |
 |                 indexed parameters                 |        :x:        |
 |                      receive                       |    :question:     |
+|         named arguments in function calls          |    :question:     |
```
Comment (DomHenderson) [inline readme.md:138]: We support things like `f({a: 3, b: 5})`, or is that not what you're referring to here?

File: readme.md [hunk 2]
```diff
@@ -135,6 +135,13 @@ Please see the list below:
...
+|                     Yul Blocks                     |    :question:     |
+|             reverts with custom errors             |    :question:     |
```
Comment (DomHenderson) [inline readme.md:140]: We support literals as error messages for reverts

File: readme.md [hunk 3]
```diff
@@ -135,6 +135,13 @@ Please see the list below:
 |                  ternary operator                  |    :question:     |
 |                 indexed parameters                 |        :x:        |
```
Comment (DomHenderson) [inline readme.md:136]: I know you didn't change this, but this should be changed to being worked on

File: readme.md [hunk 4]
```diff
@@ -135,6 +135,13 @@ Please see the list below:
...
 |                      receive                       |    :question:     |
+|         named arguments in function calls          |    :question:     |
+|                     Yul Blocks                     |    :question:     |
+|             reverts with custom errors             |    :question:     |
+|                user defined errors                 |    :question:     |
+|               function call options                |    :question:     |
```
Comment (DomHenderson) [inline readme.md:142]: Should have an example of what this is

File: readme.md [hunk 5]
```diff
@@ -135,6 +135,13 @@ Please see the list below:
...
+| member access of address object (address.balance)  |    :question:     |
+|              nested tuple expressions              |    :question:     |
```
Comment (DomHenderson) [inline readme.md:144]: to be pedantic, this should be nested tuple assignments

File: readme.md [hunk 6]
```diff
@@ -135,6 +135,13 @@ Please see the list below:
 |                  ternary operator                  |    :question:     |
 |                 indexed parameters                 |        :x:        |
 |                      receive                       |    :question:     |
+|         named arguments in function calls          |    :question:     |
+|                     Yul Blocks                     |    :question:     |
+|             reverts with custom errors             |    :question:     |
+|                user defined errors                 |    :question:     |
+|               function call options                |    :question:     |
+| member access of address object (address.balance)  |    :question:     |
```
Comment (DomHenderson) [inline readme.md:143]: this should say "e.g. address.balance" to make it clear all the address members are disallowed

**Review by DomHenderson — CHANGES_REQUESTED** [review]

**Review by DomHenderson — APPROVED** [review]

**Comment by JorikSchellekens** [comment]
No functional changes. Skipping checks

**Pull request merged**